From d82b3a639aa41ff4dff4bf4734a5c6f67411ff37 Mon Sep 17 00:00:00 2001
From: S-trace <S-trace@list.ru>
Date: Thu, 4 Feb 2016 21:41:13 +0300
Subject: [PATCH 1/3] Revert "[2/2] OpenDelta: expose rebootCustom for
 OpenDelta"

This patch breaked OpenDelta building in CM-13.0 tree:
ERROR: /home/android/cm13.0/packages/apps/OpenDelta/src/eu/chainfire/opendelta/UpdateService.java:1604: The method rebootCustom(String) is undefined for the type PowerManager
This reverts commit 2fe25d83c761676563bfd86b0ff7e723365450d5.
---
 src/eu/chainfire/opendelta/UpdateService.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/eu/chainfire/opendelta/UpdateService.java b/src/eu/chainfire/opendelta/UpdateService.java
index a2b25ae..e815ee4 100644
--- a/src/eu/chainfire/opendelta/UpdateService.java
+++ b/src/eu/chainfire/opendelta/UpdateService.java
@@ -1631,7 +1631,7 @@ OnWantUpdateCheckListener, OnSharedPreferenceChangeListener {
 
             Logger.d("flashUpdate - reboot to recovery");
 
-            ((PowerManager) getSystemService(Context.POWER_SERVICE)).rebootCustom(PowerManager.REBOOT_RECOVERY);
+            ((PowerManager) getSystemService(Context.POWER_SERVICE)).reboot("recovery");
         } catch (Exception e) {
             // We have failed to write something. There's not really anything
             // else to do at
-- 
2.11.0

